.github/workflows/build-test.yml

```diff
@@ -0,0 +1,18 @@
+name: "Docker Build"
+on:
+  push:
+    branches-ignore:
+      - "renovate/**"
+  pull_request:
+jobs:
+  test:
+    name: Dock
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@v3
+
+      - name: Build Dockerfile
+        run: |
+          docker build -f Dockerfile -t euler-rust .
```

Comment:
```suggestion
        run: docker build -t euler-rust .
```
If the Dockerfile is called Dockerfile Docker will find the file.

Reply:
I know, but I like being explicit with cmd-line stuffs. May I keep it?

Reply:
Sure, it's pretty much preference.  I slightly prefer brevity, but this fine.